### ISSUE TITLE
user_board.h removal patch is missing

### ISSUE THREAD

**Issue by ngwese** [monome/diet-asf#3]
running ```diet.sh``` currently fails because the ```patches/board.patch``` file appears to not have been added to the repository.

most likely it was accidentally omitted from 93ebfacc69913821a2ccfc122be80c48061a7b09

(i'm trying to figure out how to get usb device mode working and i think i have to add back in some services that were removed like ```sleepmgr``` in order for the usb device code to compile)

**Comment by samdoshi** [comment]
I've done the diff between the `diet-asf` produced version of `common/boards/board.h` and the one from `libavr32`. I expect that this is what `patches/board.patch` contained:

```diff
--- asf/common/boards/board.h	2016-02-04 10:39:36.000000000 +0000
+++ asf/common/boards/board.h	2016-11-16 10:52:00.000000000 +0000
@@ -372,7 +372,9 @@
 #  include "avr_simulator_uc3/avr_simulator_uc3.h"
 #elif BOARD == USER_BOARD
   // User-reserved area: #include the header file of your board here (if any).
-#  include "user_board.h"
+// changed: ezra buchla 2016-10-29
+// eliminated user_board.h, it was confusing some of the drivers.
+#  include "conf_board.h"
 #elif BOARD == DUMMY_BOARD
 #  include "dummy/dummy_board.h"
 #else
```

I'll commit the file in a few days if @catfact doesn't get round to it first.

**Comment by catfact** [comment]
oop! sorry.
https://github.com/monome/diet-asf/pull/4

**Comment by samdoshi** [comment]
Fixed in #5
